pkgs/development/libraries/libfive/default.nix

```diff
@@ -42,6 +43,10 @@ stdenv.mkDerivation {
       --replace "LIBFIVE_STDLIB_DIR=$<TARGET_FILE_DIR:libfive-stdlib>" \
                 "LIBFIVE_STDLIB_DIR=$out/lib"
 
+    substituteInPlace libfive/bind/python/CMakeLists.txt \
+      --replace ' ''${PYTHON_SITE_PACKAGES_DIR}' \
+                " $out/lib/python" \
+
```

Comment:
This may not be needed with `toPythonModule`

Reply:
This has been changed, but is still needed.